codex-rs/mcp-server/src/codex_tool_config.rs

```diff
@@ -160,6 +160,16 @@ impl CodexToolCallParam {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, JsonSchema)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
admittedly, we can name these however we want, but MCP seems to prefer `camelCase` (presumably due to its TypeScript influence since the `.ts` is the authority for the schema rather than `.json`?)